packages/shared/src/components/fields/TextField.tsx

```diff
@@ -138,7 +138,7 @@ export function TextField({
           styles.field,
         )}
       >
-        <div className="flex flex-col items-start flex-1">
+        <div className="flex flex-col items-start flex-1 max-w-full">
```

Comment:
Had to add a limiter for the width to `max-width: 100%`. The input was producing some weird additional width (shown below) probably due to the expected spin button even if it is hidden already.

![Screen Shot 2021-08-01 at 9 37 12 PM](https://user-images.githubusercontent.com/13744167/127780305-9b6d0741-4cba-4beb-b118-c73efbe531e6.png)

![Screen Shot 2021-08-01 at 9 37 58 PM](https://user-images.githubusercontent.com/13744167/127780329-bd961d2f-0359-4599-9de5-b2282209b847.png)